<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.2 (Python 3.9.1 on win32)" generated="20210104 21:53:57.076" rpa="false">
<suite id="s1" name="TC 001" source="C:\Users\Tamas\IdeaProjects\RobotAutomation2\TC_001.robot">
<test id="s1-t1" name="TC_001 - Roboy First Test case">
<kw name="Open Browser" library="SeleniumLibrary">
<doc>Opens a new browser instance to the optional ``url``.</doc>
<arguments>
<arg>${Url}</arg>
</arguments>
<msg timestamp="20210104 21:53:57.561" level="INFO">Opening browser 'firefox' to base url 'http://www.thetestingworld.com/testings'.</msg>
<msg timestamp="20210104 21:53:57.563" level="INFO">Firefox driver log is always forced to to: C:\Users\Tamas\IdeaProjects\RobotAutomation2\geckodriver-1.log</msg>
<kw name="Capture Page Screenshot" library="SeleniumLibrary">
<doc>Takes a screenshot of the current page and embeds it into a log file.</doc>
<msg timestamp="20210104 21:53:57.570" level="INFO">Cannot capture screenshot because no browser is open.</msg>
<status status="PASS" starttime="20210104 21:53:57.569" endtime="20210104 21:53:57.570"></status>
</kw>
<msg timestamp="20210104 21:53:57.575" level="FAIL">WebDriverException: Message: 'geckodriver' executable needs to be in PATH. 
</msg>
<status status="FAIL" starttime="20210104 21:53:57.561" endtime="20210104 21:53:57.575"></status>
</kw>
<status status="FAIL" starttime="20210104 21:53:57.559" endtime="20210104 21:53:57.576" critical="yes">WebDriverException: Message: 'geckodriver' executable needs to be in PATH. 
</status>
</test>
<status status="FAIL" starttime="20210104 21:53:57.080" endtime="20210104 21:53:57.578"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="TC 001">TC 001</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20210104 21:53:57.557" level="ERROR">Error in file 'C:\Users\Tamas\IdeaProjects\RobotAutomation2\TC_001.robot' on line 3: Resource file '..\Resources\Resources1.robot' does not exist.</msg>
</errors>
</robot>
